client/savePayment/html/package.json

```diff
@@ -7,7 +7,9 @@
     "build": "vite build",
     "preview": "vite preview",
     "start": "vite",
-    "format": "npx prettier . --write"
+    "format": "npx prettier . --write",
```

Comment:
Think you can drop the `npx` here.

Reply:
Great catch. Just removed `npx` from here.